src/macros.rs

```diff
@@ -1,6 +1,18 @@
 #![allow(unused_macros)]
 #![allow(unused_imports)]
 
+macro_rules! on_gba_or_unimplemented {
+  ($($token_tree:tt)*) => {
+    #[cfg(feature="on_gba")]
+    {
+      $($token_tree)*
+    }
+    #[cfg(not(feature="on_gba"))]
+    unimplemented!()
```

Comment:
Maybe consider adding a message to the `unimplemented!()` macro here. Something to indicate that this error is being hit because `on_gba` isn't enabled could possibly save someone from having to dig through the library for this line when debugging.